### ISSUE TITLE
Illegal argument exception when using command /t info

### ISSUE THREAD

**Issue by stevestech** [MyEssentials/MyTown#23]
When I use the /t info command this illegal argument exception is thrown by my permissions plugin, zPermissions, and the command fails to execute:
http://pastebin.com/tdYLGLY2

Problem started happening after I upgraded to the latest version of MyTown and its dependencies. I am running:

mcpc-plus-1.5.2-R1.1-forge738-B647.jar

mods:
forgeperms-1.5.1.3 WithoutPEX.jar
MyTown-1.5.1.11.jar
aPerf-1.5.1.1.jar

plugins:
ForgePermsCBBridge-1.1.11.jar
zPermissions-1.0.1a.jar

Another permissions-related problem occurs when I attempt to use the aPerf mod's /ap command in-game. This also just started happening after I upgraded to the latest version, and does not happen when the command is used from the console.
http://pastebin.com/LPKPaLfU

**Comment by jgoett154** [comment]
I don't seem to be able to reproduce the error when using zPermissions-1.0.1a, ForgePermsCBBridge-1.1.11, MyTown 1.5.1.11, ForgePerms 1.5.1.3, and Vault1.2.27-b349. My MCPC+ version is git-MCPC-Plus-jenkins-MCPC-Plus-Legacy-647, so should be the same as yours. Only thing I can think of is that something is wrong with Vault, I saw zPermissions has an update to 1.1 and the latest Vault may require that instead of 1.0.1a.

As for aPerf, I forgot to build a new version of it and I will do that right now.

Edit: Here should be an updated aPerf https://dl.dropboxusercontent.com/u/73334914/aPerf/aPerf-1.5.1.2.jar

**Comment by stevestech** [comment]
Thanks, I'll try and see what else could be causing problems on my end.

**Comment by stevestech** [comment]
It looks like the problem arises only when using towns from an old version of MyTown. Here are the steps I took to reproduce the problem.

1) Start by deleting the MyTown database
2) Install MyTown 1.5.1.9, ForgePerms 1.5.1.1 - Without PEX, and ForgePermsCBBridge 1.1.7
3) Create a new town and use /ta to change the mayor to an offline player, then leave the town.
4) Update to MyTown 1.5.1.11, ForgePerms 1.5.1.3 - Without PEX and ForgePermsCBBridge 1.1.11
5) Try /t info townname

By the way updating zPermissions and Vault did not have an effect.

While trying to reproduce the error, I ran into an interesting behaviour. In MyTown 1.5.1.9 when I do /t res on myself my name appears as [zPerms prefix]Sirloin[zPerms suffix], with the colour codes appearing unparsed due to MyTown expecting $[colour] not &[colour]. In MyTown 1.5.1.11 when I do /t res on myself my name appears as &fSirloin&f. I was using the same version of zPermissions and Vault, and the same permissions database, it was only the MyTown version that I was changing. Not sure if this helps at all, I just thought it might be a clue as the error made reference to getPlayerMetadata.
